src/codemagic/tools/_app_store_connect/actions/publish_action.py

```diff
@@ -55,11 +56,13 @@ class AddBuildToBetaGroupOptions:
 @dataclass
 class SubmitToTestFlightOptions:
     max_build_processing_wait: int
+    expire_previous_builds: Optional[bool] = None
```

Comment:
Isn't it pointlessly complicated default value?
```suggestion
    expire_previous_builds: bool = False
```